<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="encode_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>encode</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="encode_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>输入想调制的</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>394</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="start_record">
      <property name="text">
       <string>start record</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stop_record">
      <property name="text">
       <string>stop record and decode</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="decode_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>解调后</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
